bob-common/src/configs/node.rs

```diff
@@ -269,6 +269,8 @@ pub struct Pearl {
     disks_events_logfile: String,
     #[serde(default)]
     bloom_filter_max_buf_bits_count: Option<usize>,
+    #[serde(default = "Pearl::default_validate_data_checksum_in_index_regen")]
+    validate_data_checksum_in_index_regen: bool,
```

Comment:
Please, add this parameter to example config (https://github.com/qoollo/bob/blob/master/config-examples/node.yaml)